File: kyw0716/level0/옷가게 할인 받기.js
const getSalePrice = (price) => {
  if (price >= 500_000) return (80 / 100) * price;
  if (price >= 300_000) return (90 / 100) * price;
  if (price >= 100_000) return (95 / 100) * price;
  return price;
};

function solution(price) {
  return Math.floor(getSalePrice(price));
}

// https://school.programmers.co.kr/learn/courses/30/lessons/120818

File: sk000801/입문/옷가게 할인 받기.java
class Solution {
    public int solution(int price) {
        double price1 = 0;
        if(price >= 500000) price1 = price*0.8;
        else if(price>=300000) price1 = price*0.9;
        else if(price>=100000) price1 = price*0.95;
        else price1 = price;
        return (int) price1;
    }
}